bucket/armcord.json

```
@@ -0,0 +1,35 @@
+{
+    "version": "3.0.7",
+    "description": "A custom client designed to enhance your Discord experience while keeping everything lightweight.",
+    "homepage": "https://github.com/ArmCord/ArmCord",
+    "license": "OSL-3.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ArmCord/ArmCord/releases/download/v3.0.7/ArmCord.Setup.3.0.7.exe#/dl.7z",
+            "hash": "sha1:2097ab90eb712f69e6e2fec82f7060e0e07d61e6",
+            "pre_install": [
+                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                "Remove-Item \"$dir\\`$*\", \"$dir\\Uninst*\" -Recurse"
+            ]
+        }
+    },
```

```suggestion
    },
    "pre_uninstall": "Stop-Process -Name 'ArmCord' -ErrorAction SilentlyContinue",
```

Adding `pre_uninstall` here because ArmCord stays in tray.